src/chrome-launcher.ts

```diff
@@ -26,9 +26,12 @@ type SupportedPlatforms = 'darwin'|'linux'|'win32'|'wsl';
 
 const instances = new Set<Launcher>();
 
+type ValidPrefTypes = string|boolean|number
```

Comment:
seems like the type is much more complex here. was it intentional to limit to only a few preferences?

Reply:
No, I actually thought the `Preference` file would have a flat design. I update the type to be JSON like.